scripts/freefood.coffee

```diff
@@ -0,0 +1,52 @@
+# Description:
+#   Announce free food.
+#
+# Commands:
+#   <botname> free <food> <location>
+
+module.exports = (robot) ->
+  robot.respond /free (.+)/i, (msg) ->
```

Comment:
You can tighten up this regex to match the food & location into separate match groups, then you don't need to mangle the string as much